sandbox-templates/mdx-evaluate-create-react-app/src/App.js

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from "react";
```

Comment:
I do have significant hesitations about CRA using eval, though.
My assumption is that CRA will be chosen by folks with no clue about programming, CSS, JS. Who don’t know that backticks will have to be escaped in a backticked string for example.
Showing them that the only path is including 100kb of code to eval JavaScript on the client... Feels like a recipe for copy/paste disaster?

Reply:
Probably not.
But, CodeSandbox defaults to CRA for React examples.

> Evaluate can be used with react on its own much more cleanly?

I'm not sure I follow, the exact same approach used here would probably be used, with or without CRA?

Reply:
hmm, okay. Seems weird to me to require/use a framework when we don’t really use it.